ios/FluentUI/Avatar/Avatar.swift

```diff
@@ -435,10 +435,14 @@ public struct Avatar: View, TokenizedControlView {
         /// - Returns: hash code of string
         private static func javaHashCode(_ text: NSString) -> Int32 {
             var hash: Int32 = 0
-            for i in 0..<text.length {
-                // Allow overflows, mimicking Java behavior
-                hash = 31 &* hash &+ Int32(text.character(at: i))
-            }
+            var len = text.length - 1
+            while len >= 0 {
```

Comment:
any particular reason for while vs for?

Reply:
Not really, just thought it'd be concise. I guess I can do for loop if you have a preference?

Reply:
more to be as similar as web was my thought for ease of reference